### PR TITLE
Room rezzy error

### LINKED ISSUE #835: Data tables initializing errors
Either it's trying to instantiate it twice or it's having trouble doing it the first time.

### PR DESCRIPTION
**Pull request by billyeh**
fixes #835

### CODE REVIEW

File: ap/room_reservations/templates/room_reservations/room_reservation.html [hunk 1]
```diff
@@ -13,6 +13,7 @@
 {{ form.media }}
 <script>
   function populateForm() {
+    console.log("test")
```
Comment (jontey) [inline ap/room_reservations/templates/room_reservations/room_reservation.html]: Missed a console.log

File: ap/room_reservations/templates/room_reservations/schedule_partial.html [hunk 2]
```diff
@@ -126,32 +122,25 @@
...
-
-            // front end UI for TA to delete room reservation, consider removing
-            // {% if view.request.user|is_TA %}
-            //   row[j + 1] += `<a href="delete/${reservation.pk}">
-            //                   <button class="btn btn-danger delete_request_button" type="submit" title="Delete"/>
-            //                     <span class="glyphicon glyphicon-trash"></span>
-            //                   </button>
-            //                 </a>`
-            // {% endif %}
-            reservations.splice(k, 1);
```
Comment (jontey) [inline ap/room_reservations/templates/room_reservations/schedule_partial.html]: was this deleted by accident?

File: ap/room_reservations/forms.py [hunk 3]
```diff
@@ -21,6 +21,3 @@ def __init__(self, *args, **kwargs):
   class Meta:
     model = RoomReservation
     fields = ['group', 'date', 'start', 'end', 'room', 'group_size', 'frequency', 'reason']
-    widgets = {
-        'date': DatePicker(),
```
Comment (jontey) [inline ap/room_reservations/forms.py]: also delete the datepicker widget import on this file

File: ap/room_reservations/templates/room_reservations/schedule_partial.html (hunk 2 above)
Comment (ruthnan) [reply ap/room_reservations/templates/room_reservations/schedule_partial.html]: Does it do anything? hehe..

**Comment by billyeh** [comment]
Looks good, thanks for coding/reviewing!

**Pull request merged**